levels/mtaspiring/plants.json

```diff
@@ -0,0 +1,190 @@
+{
+  "plants": [
+    {
+      "plantID": "PLT001",
+      "category": "Fungi",
+      "name": "Beech Orange",
+      "edible": "yes"
+    },
+    {
+      "plantID": "PLT002",
+      "category": "Fungi",
+      "name": "Orange Peel Fungus",
+      "edible": "yes"
+    },
+    {
+      "plantID": "PLT003",
+      "category": "Fungi",
+      "name": "Forest Prettymouth",
+      "edible": "no"
+    },
+    {
+      "plantID": "PLT004",
+      "category": "Fungi",
+      "name": "Pixie's Parasol",
+      "edible": "yes"
+    },
+    {
+      "plantID": "PLT005",
+      "category": "Fungi",
+      "name": "Ochre Jelly Club",
+      "edible": "no"
+    },
+    {
+      "plantID": "PLT006",
+      "category": "Fungi",
+      "name": "Ruby Bonnet",
+      "edible": "no"
+    },
+    {
+      "plantID": "PLT007",
+      "category": "Fungi",
+      "name": "Dark Velvet Bolete",
+      "edible": "yes"
```

Comment:
Plants.json uses lowercase starting letter while the others use upper case. All the files should be consistent